Xamarin.Forms.Platform.Android/CollectionView/ListSource.cs

```diff
@@ -21,5 +23,24 @@ public ListSource(IEnumerable enumerable)
 				Add(item);
 			}
 		}
+
+		public void Dispose()
```

Comment:
Since we aren't actually inheriting from this anywhere right now, we could just marked it `sealed` and then we don't have to implement the `protected virtual void Dispose(bool disposing)` pattern at all.